sqlserver/README.md

```diff
@@ -1,32 +1,59 @@
-# Sqlserver Integration
+# Microsoft SQL Server Check
 
-## Overview
+# Overview
```

Comment:
That's not the standard we've been using on all of the README rewrites; we (docs team) decided some time ago on H1s.